Settings.js

```diff
@@ -222,7 +222,17 @@ function avtt_settings() {
 				{ value: true, label: "Measure", description: `When you drag a token, the distance dragged will automatically be measured. Dropping the token and picking it back up will create a waypoint in the measurement. Clicking anywhere else, or dragging another token will stop the measurement.` },
 				{ value: false, label: "Not Measuring", description: `Enable this to automatically measure the distance that you drag a token. When enabled, dropping the token and picking it back up will create a waypoint in the measurement. Clicking anywhere else, or dragging another token will stop the measurement.` }
 			],
-			defaultValue: false
+			defaultValue: true
+		},
+		{
+			name: 'showCursorToPeers',
+			label: 'Show cursor to other players',
+			type: 'toggle',
+			options: [
+				{ value: true, label: "Show Cursors", description: `When you move your cursor, others will see where your cursor is. Similarly, you will see other player's cursors.` },
+				{ value: false, label: "Don't Show Cursors", description: `You are not sharing your cursor position with other players. You are also not seeing other player's cursors.` }
+			],
+			defaultValue: true
```

Comment:
Are we able to see if the player is a DM? Maybe have this disabled by default for DMs?

Reply:
`window.DM` will show us if we're the DM. As a DM myself, I definitely want this enabled by default. However, I think you're thinking about this in a way that I hadn't. The way I built it is all or nothing, but allowing DMs to see other cursors without sending theirs is an interesting idea! We could add another setting for DMs to not send their cursor, but still receive the player's cursors. I can see some DMs not wanting to show their cursor to players. That being said, cursors are below fog of war so players won't see anything that a DM is doing in fog. So maybe that's enough. We could always add this in later if enough people ask for it, but I'm also open to including it right away. Thoughts?

Reply:
I don't see myself using the feature if it's all or nothing. I tend to have a good bit of the map revealed, but am twiddling with hidden tokens frequently that would give my players a potential insight to where things are. 
I think without the "hide dms movements" I'd have to pass on enabling this.

With that in mind, I would like to see it added in this PR but at the same time I could see your use-case would be solved so the value is there.

Up to you!